Correct error in linked guide

themelodian opened the pull request:
Because of the need to split the install.wim file when making a Windows ISO drive, the command given in the linked guide requires some correction. The rest of the guide is accurate.

themelodian committed:
Correct error in linked guide

Because of the need to split the install.wim file when making a Windows ISO drive, the command given in the linked guide requires some correction. The rest of the guide is accurate.

vit9696 commented:
You can use `/FileSize` argument instead. See this article for more details: https://www.dell.com/support/kbdoc/ru-ru/000127789/windows-10-iso-contains-wim-file-that-is-big-for-fat32-file-system?lang=en#Split. Can you update the changes?

You can also use ExFat, as long as ExFat driver is provided by OpenCore.

themelodian committed:
Update diffdisk.md

Add details on how to user ExFAT filesystem to avoid the install file splitting problem

themelodian commented:
That Dell page details how to split the `install.wim` file, not how to install the split file. I have added the ExFAT info to the guide.

vit9696 commented:
@brandazzle your "file into two .swm files" is not correct, because you may have to split the file wim file into more than two files if the original is over 8 GB. The Dell website suggests a more adequate command (through `/FileSize:4000`), which splits into as many files as necessary, that's why I want the dism command here to change.

themelodian committed:
Update diffdisk.md

Fix Apply-Image command for more than two .swm files

themelodian commented:
@vit9696 I'm sorry for the misinterpretation. Thank you for correcting my error; I have updated the command.
